changelogs/unreleased/major-sample.md

```diff
@@ -0,0 +1,11 @@
+## Sample Major Change
```

Comment:
Maybe for these also provide a real-world example? Pick out a feature we've done and write it up with actual implementation? Sometimes is helpful.